docusaurus/docs/overview/feature-model/index.mdx

```diff
@@ -1,43 +1,37 @@
 ---
-title: Data model
+title: Feature model
 ---
 
-## Key specifications
+## Key concepts
 
-Overture data is described by the [_simple feature model_](https://www.ogc.org/standard/sfa/) specified by the Open Geospatial Consortium. This means that each Overture _entity_ is represented as a single [GeoJSON](https://geojson.org/) feature: a geometric object with associated attributes and properties. For example, an _entity_ from the Overture buildings data theme might be represented in this way:
+Overture data is described using the [simple feature model](https://www.ogc.org/standard/sfa/) specified by the Open Geospatial Consortium. Each feature is represented as a geometric object with an ID and associated properties. For example, the Town Hall building in Seattle is represented in the Overture buildings dataset as [GeoJSON](https://geojson.org/) feature that looks like this:
 
 ```json
 {
-  "id": "08ba0085884a4fff0200ab2e3ad3f37c",
+  "id": "08828d542c9fffff02ffffff1953aec1",
   "type": "Feature",
   "geometry": {
     "type": "Polygon",
     "coordinates": [
-      [ [-77.036873, 38.897804],
-        [-77.036873, 38.897559],
-        [-77.03626, 38.897559],
-        [-77.03626, 38.897804],
-        [-77.036873, 38.897804] ]
+      [ [-122.3301821, 47.6090116],[-122.3301197, 47.6089409],[-122.3300642, 47.6088779],[-122.3300094, 47.6088158],[-122.329729, 47.6089282],[-122.3297466, 47.6089481],[-122.3296561, 47.6089844],[-122.3296934, 47.6090266],[-122.3297562, 47.6090978], [-122.3298113, 47.6091603],[-122.3298611, 47.6092168],[-122.3299517, 47.6091804],[-122.329963, 47.6091932],[-122.3299793, 47.6091867],[-122.3301728, 47.6091091],[-122.3301916, 47.6091016],[-122.3301714, 47.6090787],[-122.330223, 47.609058] ]
     ]
   },
   "properties": {
-    "extFoo": "I am a customer user property.",
-    "extBar": "Me too!",
     "theme": "buildings",
     "type": "building",
-    "version": 1,
-    "level": 1,
-    "updateTime": "2023-06-06T10:30:00-08:00",
-    "height": 21.34,
-    "numFloors": 4,
+    "version": 0,
+    "updateTime": "2023-01-30T00:04:13.000Z",
+    "height": 22.7,
+    "numFloors": 3,
     "class": "civic",
     "names": {
-      "primary": "Springfield Town Hall"
+      "primary": "Town Hall Seattle"
     },
     "sources": [
       {
-        "property": "",
-        "dataset": "microsoftMLBuildings,"
+        "property": "dataset",
+        "dataset": "OpenStreetMap,"
+        "recordid": "w140675988@10"
```

Comment:
Could you double-check that this isn't specified as `recordId` with an uppercase "i" in the schema?

It should be, because the current schema uses `camelCase` for property names ... Although I hope/believe we will move the whole thing to `snake_case` by March...